pkgs/development/python-modules/pyairvisual/default.nix

```diff
@@ -15,19 +15,21 @@
 
 buildPythonPackage rec {
   pname = "pyairvisual";
-  version = "5.0.9";
+  version = "2021.10.0";
   format = "pyproject";
 
-  disabled = pythonOlder "3.6";
+  disabled = pythonOlder "3.7";
```

Comment:
Not in the changelog, but: https://github.com/bachya/pyairvisual/blob/dev/pyproject.toml#L43